.github/workflows/test.yml

```diff
@@ -103,3 +103,12 @@ jobs:
           node-version: 12.x
       - run: npm ci
       - run: npm run test-cookbook
+  test-validstrings:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: use node.js v16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - run: node --experimental-modules --no-warnings --icu-data-dir ./node_modules/full-icu/ polyfill/test/validStrings.mjs
```

Comment:
The job itself is still failing due to missing timezones.json package, I'd suggest adding `run: npm ci` (which does `npm install`) before this step.

Reply:
ack, right -- done